packages/unity-interface/kernelConfigForRenderer.ts

```diff
@@ -24,6 +25,16 @@ export function kernelConfigForRenderer(): KernelConfigForRenderer {
       // tslint:disable-next-line
       typeof OffscreenCanvas !== 'undefined' && typeof OffscreenCanvasRenderingContext2D === 'function' && !WSS_ENABLED,
     network: "mainnet",
-    validWorldRanges: getWorld().validWorldRanges
+    validWorldRanges: getWorld().validWorldRanges,
+    kernelVersion: getKernelVersion(),
+    rendererVersion: getRendererVersion()
   }
 }
+
+export function getKernelVersion(): string {
+  return injectVersions({})['@dcl/kernel'] || "unknown-kernel-version"
+}
+
+export function getRendererVersion(): string {
+  return injectVersions({})['@dcl/unity-renderer'] || "unknown-renderer-version"
+}
```

Comment:
These functions are doing extra work with the injectVersions. Please look at the suggestions to prevent calling the same function twice